nhais-adaptor/edifact/incoming/parser/tests/test_creators.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+from testfixtures import compare
+import edifact.incoming.parser.creators as creators
+from edifact.incoming.models.message import MessageSegmentRegistrationDetails, MessageSegmentBeginningDetails, \
+    MessageSegmentPatientDetails
+from edifact.incoming.models.interchange import InterchangeHeader
+
+
+class TestMessage(unittest.TestCase):
+
+    def test_create_interchange_header(self):
+        expected = InterchangeHeader("SO01", "ROO5", "190429:1756")
+
+        interchange_header_dict = [

```

Comment:
Isn't this a list, rather than a dictionary?

Reply:
Actually this will be the new EdifactDict type that I can now use here too